### PR TITLE
form

### PR DESCRIPTION
**Pull request by konichek1**
- [DEMO LINK](https://konichek1.github.io/layout_html-form/)
- [TEST REPORT LINK](https://konichek1.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,182 @@
...
+      <fieldset class="field">
+        <legend>Additional info:</legend>
+        <label>
+           Comments:
+          <textarea 
+            name="comments" 
+            rows="2"
+            cols="20"
+          >
+          </textarea>
```
Comment (loralevitska) [inline src/index.html]: Fix cursor 
image

```suggestion
          ></textarea>
```

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,182 @@
...
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button 
+        type="submit"
+        class="about"
+       >Submit
+      </button>
```
Comment (loralevitska) [inline src/index.html]: Fix all cases if present

```suggestion
      <button 
        type="submit"
        class="about"
      >
        Submit
      </button>
```

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,182 @@
...
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+          Do you love cats?
+        <label>
+          <input 
+           type="radio" 
+           name="loveCats" 
+           value="y"
```
Comment (loralevitska) [inline src/index.html]: Use more meaningful naming for values. `yes`?!
Fix all cases

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job!
Let's improve

**Comment by konichek1** [comment]
- [DEMO LINK](https://konichek1.github.io/layout_html-form/)
- [TEST REPORT LINK](https://konichek1.github.io/layout_html-form/report/html_report/)

**Pull request closed without merging**